generators/app/templates/src/redux/store.ejs

```diff
@@ -7,15 +7,14 @@ import {
 } from 'react-navigation-redux-helpers';
 import { fetchMiddleware, configureMergeState } from 'redux-recompose';
 <%_ if(features.reduxpersist) { _%>
-  import AsyncStorage from '@react-native-community/async-storage';
-  import { persistReducer } from 'redux-persist';
-  import {
-    seamlessImmutableReconciler,
-    seamlessImmutableTransformCreator
-  } from 'redux-persist-seamless-immutable';
-  <%_ } _%>
+import AsyncStorage from '@react-native-community/async-storage';
+import { persistReducer } from 'redux-persist';
+import {
+  seamlessImmutableReconciler,
+  seamlessImmutableTransformCreator
+} from 'redux-persist-seamless-immutable';
+<%_ } _%>
 
-import Navigator from '@screens';
```

Comment:
Why did you move this line?

Reply:
when I apply the esLint of react-native-community, it obligate me to sort imports in this file